packages/ui/src/council/types/PastCouncil.ts

```diff
@@ -52,8 +52,8 @@ export const asPastCouncilWithDetails = (
   return {
     ...asPastCouncil(councilFields),
     totalSpent: getTotalSpent(spendingEvents),
-    totalMissedRewards: councilFields.councilMembers.reduce((a, b) => a.addn(b.unpaidReward), BN_ZERO).neg(),
-    totalPaidRewards: councilFields.councilMembers.reduce((a, b) => a.addn(b.accumulatedReward), BN_ZERO),
+    totalMissedRewards: councilFields.councilMembers.reduce((a, b) => a.addn(Number(b.unpaidReward)), BN_ZERO).neg(),
+    totalPaidRewards: councilFields.councilMembers.reduce((a, b) => a.addn(Number(b.accumulatedReward)), BN_ZERO),
```

Comment:
```suggestion
    totalMissedRewards: councilFields.councilMembers.reduce((a, b) => a.add(new BN(b.unpaidReward)), BN_ZERO).neg(),
    totalPaidRewards: councilFields.councilMembers.reduce((a, b) => a.add(new BN(b.accumulatedReward)), BN_ZERO),
```
It would be better to convert these straight into BNs just in case they get to big.